### DISCUSSION TITLE
执行python run.py 命令，出现了卡死的情况

### DISCUSSION THREAD

**Discussion by senwang01** [open-compass/opencompass#940] [Q&A]
模型为llama-7b, 数据集已经下载到本地，在A800上执行下面命令：
python run.py --datasets ceval_ppl   mmlu_ppl --hf-path llama-7b-convert-hf --model-kwargs device_map='auto' --tokenizer-kwargs padding_side='left' truncation='left' use_fast=False --max-out-len 100 --max-seq-len 2048 --batch-size 8 --no-batch-padding --num-gpus 1    
出现了卡死的情况，该如何处理了
![image](https://github.com/open-compass/opencompass/assets/22807449/03b79f78-004b-4424-a0dc-9cf0eaade8f5)

**Comment by bittersweet1999** [comment]
You can check and provide your logs, it default in 'outputs' folder

**Reply by senwang01** [reply]
执行命令 logs/infer/opencompass.models.huggingface.HuggingFace_data_llama-7b-convert-hf/lukaemon_mmlu_moral_scenarios.out查看输出：
![image](https://github.com/open-compass/opencompass/assets/22807449/12bf26dc-4b97-4ca5-a548-7ad0e0e305fd)

目前已经running了1个半小时，这种情况正常吗

**Comment by bittersweet1999** [comment]
It seems normal. If you feel it's taking too long, considering the large number of subsets in the ceval dataset, you might want to test fewer subsets. Alternatively, if you have sufficient GPU resources, you could consider increasing the number of partitions.
